frontend/controller/backend.js

```diff
@@ -111,8 +118,22 @@ sbp('sbp/selectors/register', {
     return fetch(`${sbp('okTurtles.data/get', 'API_URL')}/latestHash/${contractID}`)
       .then(handleFetchResult('text'))
   },
-  'backend/translations/get': (language: string) => {
-    return fetch(`${sbp('okTurtles.data/get', 'API_URL')}/translations/get/${language}`)
-      .then(handleFetchResult('json'))
+  /**
+   * Fetches a JSON object containing translation strings for a given language.
+   *
+   * @param language - A BPC-47 language tag like the value
+   * of `navigator.language`.
+   *
+   * @see The 'translations/init' SBP selector in `~view-utils/translations.js`.
+   */
+  async 'backend/translations/get' (language: string): Promise<?JSONObject> {
+    // The language code is usually the first part of the language tag.
+    const [languageCode] = language.toLowerCase().split('-')
+    const languageFileName = languageFileMap.get(languageCode) || ''
```

Comment:
What's the need for ` || ''` at the end here?

Reply:
It allows `languageFileName` to always be a string by disallowing the `undefined` value.
This can be removed, I just prefer to avoid mixing types when it's not necessary.

Reply:
I see. It's a quirk of JavaScript that `''` is falsey. In other languages the `if` would pass... so that's the only reason why I'd consider removing it. Not a strong reason, so can leave as is.

Reply:
We can be even more explicit by writing `languageFileName === ''` in the `if` clause rather than relying on implicit boolean conversion.

Reply:
Totally up to you. I'm ok with it either way, just was curious about why you didn't rely on the `undefined`.